market/market/coingecko/coingecko.go

```diff
@@ -85,6 +91,13 @@ func (m *Market) normalizeTicker(price coingecko.CoinPrice, provider string) (ti
 	return
 }
 
+func isSuitableTicker(marketCap, totalVolume, circulatingSupply float64) bool {
```

Comment:
out of the signature you're only using one value, `marketCap`, why keep the others?

Reply:
> @EnoRage ultimately I think the filter logic should not be part of `normalizeRates`. How do you feel about pulling the logic out of `normalizeRates` and making a dedicated `filter` function?

The only disadvantage of these approach is that we will need to walk through the array again, what is not great. However the codebase must be as much readable as possible, so it is better to use your approach